### DISCUSSION TITLE
Support for uvm_reg like Systemverilog's UVM

### DISCUSSION THREAD

**Discussion by jiyong00** [pyuvm/pyuvm#104] [Q&A] [answered]
Does it support uvm_reg like Systemverilog's UVM?
If not, you have plan to support it ?

**Comment by raysalemi** [comment, marked as answer]
> Does it support uvm_reg like Systemverilog's UVM?
> 
> If not, you have plan to support it ?

There is a project underway to support it. Two folks have contributed to that effort.

**Reply by jiyong00** [reply]
Thank you for your contribution.
